src/test/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncServiceTest.java

```diff
@@ -414,11 +414,33 @@ void shouldDeleteProgrammeMemberships(Operation operation) {
     service.syncRecord(record);
 
     verify(restTemplate)
-        .patchForObject(anyString(), eq(null), eq(Object.class), eq("programme-membership"),
+        .delete(anyString(), eq("programme-membership"),
             eq("personIdValue"));
     verifyNoMoreInteractions(restTemplate);
   }
 
+  @ParameterizedTest(
+      name = "Should not delete when operation is {0} and table is not "
+          + "ProgrammeMembership")
```

Comment:
Needs to be modified. Currently e.g.:
```
Should not delete when operation is ContactDetails and table is not ProgrammeMembership
```
How about
```
@ParameterizedTest(name = "Should not delete when operation is DELETE and table is {0}")
```

Reply:
Good catch. I've also rebased on Ed's work, so there should be no conflicts.